src/caffe/layers/cudnn_conv_layer.cpp

```diff
@@ -2,25 +2,23 @@
 #include <algorithm>
 #include <vector>
 
+#include "caffe/filler.hpp"
 #include "caffe/layers/cudnn_conv_layer.hpp"
+#include "caffe/util/gpu_memory.hpp"
+#include "caffe/util/im2col.hpp"
+#include "caffe/util/math_functions.hpp"
 
-namespace caffe {
 
-// Set to three for the benefit of the backward pass, which
-// can use separate streams for calculating the gradient w.r.t.
-// bias, filter weights, and bottom data for each group independently
-#define CUDNN_STREAMS_PER_GROUP 3
+namespace caffe {
 
 /**
```

Comment:
By the way, this whole file has a pretty messed up style... everything is indented inside the `namespace` (it shouldn't be), and there are lots of unneeded blank lines. (Caffe style is generally pretty compact wrt blank lines, which I can see arguments either way, but there's no reason to be inconsistent here.)

If you do fix these style issues, however, please do so in a separate commit.